the-telephone-app-fe/src/components/ShowResult/ShowResult.css

```diff
@@ -209,4 +209,78 @@
 
 .sr-next-button:hover {
     background-color: #e5b48c;
-}
\ No newline at end of file
+}
+
+.chat-bubble-right {
+    background-color: var(--secondary-color);
+    padding: 8px 14px;
+    -webkit-border-radius: 20px;
+    -webkit-border-top-right-radius: 2px;
+    -moz-border-radius: 20px;
+    -moz-border-radius-bottomleft: 2px;
+    border-radius: 20px;
+    border-top-right-radius: 2px;
+    display: inline-block;
+}
+
+.chat-bubble-left {
+    background-color: var(--secondary-color);
+    padding: 8px 14px;
+    -webkit-border-radius: 20px;
+    -webkit-border-top-left-radius: 2px;
+    -moz-border-radius: 20px;
+    -moz-border-radius-bottomleft: 2px;
+    border-radius: 20px;
+    border-top-left-radius: 2px;
+    display: inline-block;
+}
```

Comment:
Don't repeat code, use a general class to put what this two class shared (like chat-bubble) and leave only what is different from each other here.